src/interfaces.cairo

```diff
@@ -9,10 +9,49 @@ trait IFunctionVerifier<TContractState> {
 
 #[starknet::interface]
 trait ISuccinctGateway<TContractState> {
+ /// Returns the fee vault address.
+        ///
+        /// # Returns
+        ///
+        /// * The fee vault address.
```

Comment:
Please fix the formatting for all the added comments

Reply:
> You should be able to run `scarb fmt` for formatting

Done and pushed. Thank you.